swiftshield-Sources/AutomaticSwiftShield.swift

```diff
@@ -81,6 +81,13 @@ class AutomaticSwiftShield: Protector {
                 }
                 let name = data.name
                 let usr = data.usr
+                let kind = data.kind
+                
+                if let type = self.sourceKit.referenceType(kind: kind), type == .enum && name.hasSuffix("CodingKeys") {
+                    obfuscationData.excludedEnums.insert(self.usrWithoutSuffix(usr))
```

Comment:
Everything looks good now, but I'm just wondering if we need to do `self.usrWithoutSuffix(usr)` here. The enum usr is already the stripped one, isn't it?